docs/src/main/paradox/overview.md

```diff
@@ -14,17 +14,17 @@ This plugin requires **Akka $akka.version$** or later. See [Akka's Binary Compat
   group=com.typesafe.akka
   artifact=akka-persistence-cassandra_$scala.binary.version$
   version=$project.version$
-  symbol=AkkaVersion
+  symbol=pekkoVersion
```

Comment:
Why did this get changed to lower case, can it be `PekkoVersion` to make sure its consistent with the other projects?  (note there are other cases of this).

Reply:
in that incubator-pekko project, we use PekkoVersion in the `.md` files - eg https://github.com/apache/incubator-pekko/blob/main/docs/src/main/paradox/persistence-fsm.md

Reply:
right, I'll change it to PekkoVersion then